packages/base/src/signal-manager.ts

```diff
@@ -12,7 +13,8 @@ export const Signals = {
     EXPERIMENT_OPENED: 'experiment opened',
     EXPERIMENT_CLOSED: 'experiment closed',
     EXPERIMENT_SELECTED: 'experiment selected',
-    TOOLTIP_UPDATED: 'tooltip updated'
+    TOOLTIP_UPDATED: 'tooltip updated',
+    EVENT_CLICKED: 'Event clicked'
```

Comment:
Event -> event: should lower case. I wonder if we will re-use this signal for other selections changes (e.g. chart to table). So, maybe we call it "selection changed".